packages/template-typescript-minimal/jest.config.js

```diff
@@ -16,7 +16,5 @@ module.exports = {
             statements: 0
         }
     },
-    collectCoverageFrom: [
-        'app/**'
-    ]
+    collectCoverageFrom: ['app/**']
```

Comment:
is the format intentional?

Reply:
yep. result of `npm run format`

Reply:
Dumb question, how does it different than running `npm lint fix`?

Reply:
we created a npm script for fixing the lint issues. it's equivalent to `npm run lint -- --fix`